src/main/webapp/team.html

```diff
@@ -1,30 +1,244 @@
+        </div>
     </div>
-  </body>
+
+    <!-- MODAL CREATE TASK-->
+    <div class="modal fade " id="createtask" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog ">
+            <div class="modal-content ">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Create a Task</h5>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+
+                    <form name="newTask" action="/create-task" method="POST" onsubmit="addUserNForm(); return validateForm(); " >
+                        
+                            <input type="hidden" name="userN" class="TaskUserName" type="text"/>
+                        
+                        <div class="mb-3">
+                            <p>Task Title:</p>
+                            <input type="text" name="title" style="width: 100%;"></input>
+                        </div>
+                        <div class="mb-3">
+                            <p>Description:</p>
+                            <textarea name="desc" style="width: 100%;"></textarea>
+                        </div>
+                        <div class="mb-1">
+                            <label>
+                                <p>Time Goal (Minutes): </p>
+                            </label>
+                            <input type="number" min="0" style="width: 4vw;" oninput="this.value = Math.abs(this.value)"
+                                name="time"></input>
+                        </div>
+                        <!-- COLLABORATE STATUS
+                        <div class="mb-3">
+                            <label>
+                                <p>Available to collaborate?</p>
+                            </label>
+                            <input class="ms-3 form-check-input" type="radio" name="colla" id="flexRadioDefault1" value="yes">
+                            <label class="form-check-label" for="flexRadioDefault1">
+                                YES
+                            </label>
+                            <input class="ms-3 form-check-input" type="radio" name="colla" id="flexRadioDefault2" value="no" checked>
+                            <label class="form-check-label" for="flexRadioDefault2">
+                                NO
+                            </label>
+                        </div>
+                        -->
+                        <div class="modal-footer">
+                            <button type="reset" class="btn btn-danger" data-bs-dismiss="modal">Cancel</button>
+                            <button type="submit" class="btn btn-success">Create Task!</button>
+                        </div>
+                    </form>
+                </div>
+
+            </div>
+
+        </div>
+
+    </div>
+
+    <!-- MODAL END TASK (NOT IN USE)
```

Comment:
Should we remove the unused commented out parts?

Reply:
Yes... Done